### DISCUSSION TITLE
Multiline Support to Node

### DISCUSSION THREAD

**Discussion by TheDemonHuB** [eclipse-glsp/glsp#831] [Q&A]
![image](https://user-images.githubusercontent.com/57061430/208834757-80ee2096-5b63-47f2-bd61-8896d2013ec1.png)
how can i enable multiline support to nodes ?
is there any predefined function?

**Comment by tortmayr** [comment]
Hi @TheDemonHuB,

SVG has no builtin support for line wrapping. As a workaround you can use a `ForeignObjectElement`. 
This allows you to render HTML elements like a multiline textbox inside your diagram.

Please have a look at the GLSP website documentation.
In the ['Rendering & Styling](https://www.eclipse.org/glsp/documentation/rendering/#default-views) section there is an example (including code snippets) that demonstrates how to use the `ForeignObjectView` for a multiline text node.
